Source/Rows/StepperRow.swift

```diff
@@ -8,88 +8,154 @@
-
-        addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-[t]-[v]-[s]-|", options: .alignAllCenterY, metrics: nil,
-                                                      views: ["s": stepper, "v": valueLabel, "t": textLabel]))
-        addConstraint(NSLayoutConstraint(item: stepper, attribute: .centerY, relatedBy: .equal, toItem: self, attribute: .centerY,
-                                         multiplier: 1.0, constant: 0))
-        addConstraint(NSLayoutConstraint(item: valueLabel, attribute: .centerY, relatedBy: .equal, toItem: stepper, attribute: .centerY,
-                                         multiplier: 1.0, constant: 0))
-        addConstraint(NSLayoutConstraint(item: textLabel, attribute: .centerY, relatedBy: .equal, toItem: stepper, attribute: .centerY,
-                                         multiplier: 1.0, constant: 0))
-    }
-
-    required public init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
+  
+  private var awakeFromNibCalled = false
+  
+  @IBOutlet open weak var titleLabel: UILabel!
+  @IBOutlet open weak var valueLabel: UILabel!
+  @IBOutlet open weak var stepper: UIStepper!
+  
+  open var formatter: NumberFormatter?
+  
+  public required init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: .value1, reuseIdentifier: reuseIdentifier)
+    
+    NotificationCenter.default.addObserver(forName: UIContentSizeCategory.didChangeNotification, object: nil, queue: nil) { [weak self] _ in
+      guard let me = self else { return }
+      if me.shouldShowTitle {
+        me.titleLabel = me.textLabel
+        me.valueLabel = me.detailTextLabel
+        me.setNeedsUpdateConstraints()
+      }
     }
-
-    open override func setup() {
-        super.setup()
-        selectionStyle = .none
-        stepper.addTarget(self, action: #selector(StepperCell.valueChanged), for: .valueChanged)
+  }
+  
+  deinit {
+    guard !awakeFromNibCalled else { return }
+    NotificationCenter.default.removeObserver(self, name: UIContentSizeCategory.didChangeNotification, object: nil)
+  }
+  
+  required public init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+    awakeFromNibCalled = true
+  }
+  
+  open override func setup() {
+    super.setup()
+    if !awakeFromNibCalled {
+      let title = textLabel
+      textLabel?.translatesAutoresizingMaskIntoConstraints = false
+      textLabel?.setContentHuggingPriority(UILayoutPriority(rawValue: 500), for: .horizontal)
+      self.titleLabel = title
+      
+      let value = detailTextLabel
+      value?.translatesAutoresizingMaskIntoConstraints = false
+      value?.setContentHuggingPriority(UILayoutPriority(500), for: .horizontal)
+      value?.adjustsFontSizeToFitWidth = true
+      value?.minimumScaleFactor = 0.5
+      self.valueLabel = value
+      let font = self.valueLabel.font!
+      self.valueLabel.font = UIFont.monospacedDigitSystemFont(ofSize: font.pointSize, weight: font.weight)
```

Comment:
I don't think it is necessary to set this font here, and if then prefer `systemFont(ofSize:)`.

Reply:
ok sure. As i was copying from SliderRow first, the numbers were positioned after the stepper. There, it shifted on ever digit change. There the monospaced digit font was a great fit.